docs/source/best-practices/homequota.md

```diff
@@ -31,6 +31,18 @@ Please be mindful of the `/scratch` purge policy, which can be found on the [Res
 :::
 
 ## Cleaning Directories
+### Local
+
+We advise against using 'pip install' to install packages outside of a conda environment or python virtual environment (for example, while in a JupyterLab Notebook or interactive python session). These installations are placed in your `.local` directory, adding to your `/home` quota. Additionaly, the presence of different packages in `.local` can have a negative impact on the function of applications on the OOD. Please ensure all the packages you need are installed in a conda or virtual python environment.
+
+If there are no activly running processes the entire `.local` directory can be deleted or individual packages can be removed usually from within: `/home/username/.local/lib/pythonXX/site-packages`
```

Comment:
I agree with this comment but this might lead to tickets if this breaks other notebooks and items. I might suggest a mv command to get things working and rebuild and delete when it is up and running again so nothing is lost. 

Also might be handy to suggest if JupyterLab is crashing, the output.log file in the session ID will often inform you on what package is crashing the environment. This might just be something for internal notes on how to diagnosis it for the users but it might be nice to include for the users who want to figure out what was breaking their environments.

Reply:
This is a good idea. I've revised it to move and added a line showing how to move to --off.